src/organization/configuration/identity-provider/org-settings-identity-provider-graviteeio-am/org-settings-identity-provider-graviteeio-am.component.html

```diff
@@ -15,4 +15,65 @@
     limitations under the License.
 
 -->
-gravitee
+<mat-card class="identity-provider__card">
+  <h2>Configuration</h2>
+
+  <mat-form-field appearance="fill" class="identity-provider__card__form-field">
```

Comment:
For consistency in this file ;)

```suggestion 
  <mat-form-field class="identity-provider__card__form-field" appearance="fill">
```

Same for some of the next ones ⬇️